pinot-spi/src/main/java/org/apache/pinot/spi/ingestion/batch/IngestionJobLauncher.java

```diff
@@ -36,24 +41,34 @@
 
   public static final Logger LOGGER = LoggerFactory.getLogger(IngestionJobLauncher.class);
 
-  private static final String USAGE = "usage: [jobSpec.yaml]";
+  private static final String USAGE = "usage: [jobSpec.yaml] [template_key=template_value]...";
```

Comment:
What's the delimiter between two `[template_key=template_value]`? (Assuming `...` indicates there's more than one of those.)

Reply:
It will be space, the arg4j parse it into an array. Adding example in PR description: 
```
bin/pinot-admin.sh LaunchDataIngestionJob  \
-jobSpecFile ingestionJobSpec.yaml \
-values year=2020 month=05 day=06 hour=07
```